lib/pagerduty.rb

```diff
@@ -52,13 +51,13 @@ def initialize(service_key)
   #
   # @return [PagerdutyIncident] The triggered incident.
   #
-  # @raise [PagerdutyException] If PagerDuty responds with a status that is not
+  # @fail [PagerdutyException] If PagerDuty responds with a status that is not
```

Comment:
hmm fair point, missed that.  I can revert to raise and put a rubocop exclude in